roles/openshift_node_group/defaults/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+openshift_node_groups:
+- type: master
```

Comment:
How are you using `type`?  I would prefer `name` - there is no such thing as `type`.

Reply:
Node groups aren't special, but they are identified by name.  Targeting pods and workloads to node groups is done by labels.  Multiple node groups can have the same labels.

Reply:
I was using it like this in main.yml:
```
    openshift_node_group_name: "{{ openshift_node_group_prefix}}{{ node_group.type }}"
```
I can switch to name if that's preferred.

Reply:
Yeah, name would be preferred.

Reply:
Switched.  Can't test because images aren't available from the docker registries.